charts/rancher-monitoring/v0.0.3/charts/grafana/dashboards/w_prometheus_stats-dashboard.json

```diff
@@ -0,0 +1 @@
ax":null,"min":null,"show":true}],"yaxis":{"align":false,"alignLevel":null}},{"aliasColors":{"Chunks":"#1F78C1","Chunks to persist":"#508642","Max chunks":"#052B51","Max to persist":"#3F6833"},"bars":false,"dashLength":10,"dashes":false,"datasource":"RANCHER_MONITORING","editable":true,"error":false,"fill":1,"gridPos":{"h":7,"w":8,"x":16,"y":25},"id":22,"legend":{"avg":false,"current":false,"max":false,"min":false,"show":true,"total":false,"values":false},"lines":true,"linewidth":1,"links":[],"nullPointMode":"null","percentage":false,"pointradius":5,"points":false,"renderer":"flot","seriesOverrides":[],"spaceLength":10,"stack":false,"steppedLine":false,"targets":[{"expr":"round(increase(prometheus_rule_group_iterations_missed_total{job=\"$job\", namespace=\"$namespace\"}[10m]))","format":"time_series","intervalFactor":2,"legendFormat":"Missed","metric":"prometheus_local_storage_memory_chunkdescs","refId":"B","step":10},{"expr":"round(increase(prometheus_rule_evaluation_failures_total{job=\"$job\", namespace=\"$namespace\"}[10m]))","format":"time_series","intervalFactor":2,"legendFormat":"Failed","metric":"prometheus_local_storage_memory_chunkdescs","refId":"C","step":10}],"thresholds":[],"timeFrom":null,"timeRegions":[],"timeShift":null,"title":"Rule Group Evaluation Problem","tooltip":{"msResolution":false,"shared":true,"sort":0,"value_type":"individual"},"type":"graph","xaxis":{"buckets":null,"mode":"time","name":null,"show":true,"values":[]},"yaxes":[{"decimals":0,"format":"short","label":null,"logBase":1,"max":null,"min":"0","show":true},{"format":"short","label":null,"logBase":1,"max":null,"min":null,"show":true}],"yaxis":{"align":false,"alignLevel":null}}],"title":"Query Engine","type":"row"}],"refresh":"1m","schemaVersion":16,"style":"dark","tags":[],"templating":{"list":[{"allValue":null,"current":{"text":"$job","value":"$job"},"datasource":"RANCHER_MONITORING","definition":"query_result(prometheus_tsdb_head_samples_appended_total)","hide":2,"includeAll":false,"label":null,"multi":false,"name":"job","options":[],"query":"query_result(prometheus_tsdb_head_samples_appended_total)","refresh":2,"regex":"/.*job=\"([^\"]+)/","skipUrlSync":false,"sort":1,"tagValuesQuery":"","tags":[],"tagsQuery":"","type":"query","useTags":false},{"allValue":null,"current":{"text":"cattle-prometheus","value":"cattle-prometheus"},"datasource":"RANCHER_MONITORING","definition":"query_result(up{job=~\"$job\"})","hide":0,"includeAll":false,"label":"Namespace","multi":false,"name":"namespace","options":[],"query":"query_result(up{job=~\"$job\"})","refresh":2,"regex":"/.*namespace=\"([^\"]+).*/","skipUrlSync":false,"sort":0,"tagValuesQuery":"","tags":[],"tagsQuery":"","type":"query","useTags":false}]},"time":{"from":"now-5m","to":"now"},"timepicker":{"refresh_intervals":["5s","10s","30s","1m","5m","15m","30m","1h","2h","1d"],"time_options":["5m","15m","1h","6h","12h","24h","2d","7d","30d"]},"timezone":"browser","title":"Prometheus Stats","uid":"4hEg-4eiz","version":1}
```

Comment:
can you explain what these giant json blobs are?

Reply:
Please remove the change for grafana dashboard. It's nice to have. But we should not do this for now. If you want to debug for users in 2.2.2, you can just ask users to import the grafana dashboard then.